src/tera.rs

```diff
@@ -638,6 +675,22 @@ mod tests {
     use context::Context;
     use serde_json::{Map as JsonObject, Value as JsonValue};
 
+    fn escape_c_string(input: &str) -> String {
```

Comment:
when would this be used? I think this doesn't need to be in Tera directly

Reply:
`escape_c_string` is in `tera::tests` module, so it is not visible in production code.

Reply:
I moved `escape_c_string` into test function.

Reply:
I missed that ;( sorry